stock_management/tmp_win_utils.py

```diff
@@ -0,0 +1,36 @@
+from stock_management.layouts import get_test_layout
```

Comment:
The package `pytest`, by default, looks for all the `.py` that starts with `test` and consider them as tests to be run during the CI (ask CGPT).

Hence, you should never call your script "test_<smt>" if it is not a proper test in the tests folder. Otherwise, pytest will try to run it as a test, and as you see https://github.com/francescocarpanese/stock_management/actions/runs/6603773133/job/17937243641?pr=45 your automatic testing will fail